plugin/src/main/java/org/opensearch/ml/model/MLModelManager.java

```diff
@@ -392,9 +391,6 @@ private void indexRemoteModel(MLRegisterModelInput registerModelInput, MLTask ml
         String taskId = mlTask.getTaskId();
         FunctionName functionName = mlTask.getFunctionName();
         try (ThreadContext.StoredContext context = client.threadPool().getThreadContext().stashContext()) {
-            mlStats.getStat(MLNodeLevelStat.ML_REQUEST_COUNT).increment();
-            mlStats.createCounterStatIfAbsent(functionName, REGISTER, ML_ACTION_REQUEST_COUNT).increment();
```

Comment:
This line is to track how many register requests on function level. By removing this, can we still track that?

Reply:
Yeah, because we are tracking this in the parent function `registerMLModel`